src/NuGet.Services.EndToEnd/Support/Clients/GalleryClient.cs

```diff
@@ -108,10 +108,13 @@ public async Task<IList<string>> AutocompletePackageVersionsAsync(string id, boo
             return await _httpClient.GetJsonAsync<List<string>>(uri, logger);
         }
 
-        public async Task PushAsync(Stream nupkgStream, ITestOutputHelper logger)
+        public async Task PushAsync(Stream nupkgStream, ITestOutputHelper logger, bool isSymbolsPackage = false)
         {
             var galleryEndpoint = await GetGalleryUrlAsync(logger);
-            var url = $"{galleryEndpoint}/api/v2/package";
+            var url = isSymbolsPackage
+                ? $"{galleryEndpoint}/api/v2/symbolpackage"
```

Comment:
Should we look up the push URLs from the service index? This is what the client does, it'd be a "stronger" test.

Reply:
I agree, but I would rather not make unrelated changes in this PR.